lib/rubocop/util.rb

```diff
@@ -12,5 +12,13 @@ def self.silence_warnings
     ensure
       $VERBOSE = old_verbose
     end
+
+    def self.display_width(string)
+      if string.ascii_only?
+        string.size
+      else
+        Unicode::DisplayWidth.of(string)
+      end
+    end
```

Comment:
This change should probably be upstreamed to the `unicode-display_width` gem. But I would suggest to have it here, until that is done.

Reply:
Is this safe to do here? For certain characters, these wouldn't be equal, e.g.:
```ruby
s = "\x00"
s.ascii_only?                # true
s.size                       # 1
Unicode::DisplayWidth.of(s)  # 0
```

Reply:
Correct, thanks. I already made that check in the linked PR - https://github.com/janlelis/unicode-display_width/pull/26.